conan/cli/printers/graph.py

```diff
@@ -99,7 +99,7 @@ def _format_requires(title, reqs_to_print):
             return
         output.info(title, Color.BRIGHT_YELLOW)
         for pref, (status, remote) in sorted(reqs_to_print.items(), key=repr):
-            if remote is not None:
+            if remote is not None and status != BINARY_SKIP:
```

Comment:
Is this a fix related to the PR?

Reply:
yes, otherwise the checks ``c.assert_listed_binary({"base/1.0": ("62e589af96a19807968167026d906e63ed4de1f5", "Skip"),`` failed as it was printing ``Skip (default)`` which doesn't make much sense